### PR TITLE
Rename cf-k8s-secrets repo to korifi-secrets

### PR DESCRIPTION
**Pull request by gnovv**
This PR is to rename our cf-k8s-secrets repo used for CI to match the new project name.
This is required so that the repo is not automatically recreated when we rename it in the Github UI.

@emalm could you please help us coordinate and expedite this change

### CODE REVIEW

**Review by emalm — APPROVED** [review]

**Comment by emalm** [comment]
LGTM! Requesting review from the other TOC members. Once we have TOC approval, let's coordinate merging this PR with the actual rename of the repo so that Peribolos doesn't wig out.

Also, could you or someone else in the CF on K8s WG please make a separate PR to add https://github.com/cloudfoundry/korifi-secrets and https://github.com/cloudfoundry/korifi-ci to the WG charter document? Then we can get TOC approval on that and merge it once all the renames of existing repos are done.

**Comment by gnovv** [comment]
@emalm I created this PR to update our charter:
https://github.com/cloudfoundry/community/pull/258

**Review by stephanme — APPROVED** [review]

**Review by georgethebeatle — APPROVED** [review]
LGTM

File: org/cloudfoundry.yml
Comment (georgethebeatle) [inline org/cloudfoundry.yml:6168]: This one was not renamed, since it is secret anyway and makes sense to be named after the working group

**Review by gcapizzi — COMMENTED** [review]
It looks like we still haven't agreed on wether this rename should happen - I agree that we could keep the existing name and use the repo for all WG secrets, not just Korifi ones (although we might not have any other secrets for now).

**Comment by gnovv** [comment]
@gcapizzi that sounds good to me! I will close this PR and tweak the charter one
Appreciate everyone's input!

**Pull request closed without merging**
